### ISSUE TITLE
ipkg branch: Show holes is empty, Add Clause shows error

### ISSUE THREAD

**Issue by NobbZ** [idris-hackers/atom-language-idris#53]
I'm on the `ipkg`-branch (commit 26f6b82 to be exact) and even when typechecking does find all modules now, I have some other trouble, I didn't had on release:

* When trying to add a clause I get a pane labeled "Idris Errors". The contentarea of the pane is ": openFile: does not exist (No such file or directory)".
* When I try to use the show holes function, I get a pane labeled "Idris: Holes". Contentarea shows just a darkgray background. Just as release looks like, but without anything written into the box.

This happens as well when I am in a directory with an `ipkg` file as well as without.

Minimal example that shows the problem I have:

```idr
module Foo

import Data.Vect

bar : Vect n a -> Vect m a -> Vect (n + m) a
--bar xs ys = ?bar_rhs
```

I'd expect that something similar to the commented line would be created when using add clause. Also when uncommenting it, I'd expect to see typeinfo about `?bar_rhs` in the bottom pane.

Also I am using Idris from git, currently it is (exactly that binary is also configured in atom):

```
$ /home/nobbz/projects/idris-source/.cabal-sandbox/bin/idris --version
0.9.18.1-git:8205c13
```

But I don't think idris version should be an issue, it did work before with latest release.

**Comment by archaeron** [comment]
Hey

thanks a lot for your bug report. It is very much appreciated!

I did reproduce the holes bug. Haven't yet tried the other one.

**Comment by archaeron** [comment]
I finally found the error and it's really stupid :(

**Comment by archaeron** [comment]
@NobbZ It would be cool if you'd find the time and check again if it's working for you now after a pull of this branch. :)

**Comment by NobbZ** [comment]
I'll check during the week, but it will take some time, I currently have to
do a lot of work under windows, but my hole idris stuff is installed on the
Linux partition.

archaeron <notifications@github.com> schrieb am Mo., 07.09.2015, 1:30:

**Comment by archaeron** [comment]
no worries, I have the same probllem. That's why it took me so long to find the bug.

**Comment by archaeron** [comment]
should be fixed in v0.3.0
